CSS/Components/list.css

```diff
@@ -0,0 +1,29 @@
+.list-container {
+    width: 15rem;
+    height: max-content;
+    background-color: var(--white-color);
+    padding: 0.5rem 0;
+  }
+  .list-label {
+    padding: 1rem;
+    font-size: 1.5rem;
+    border-bottom: 1px solid black;
+  }
+  .list {
+    padding-inline-start: 0;
+  }
+  .list li {
+    padding: 0.5rem 1rem;
+    font-weight: 400;
+  }
+  .list li:hover {
+    color: var(--primary-color);
+    cursor: pointer;
+  }
+  .stackedlist li {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
```

Comment:
do create utility classes for flex and justify combinations, as it'll help in upcoming projects

Reply:
noted Sam !